pkg/storage/chunk/client/gcp/bigtable_object_client.go

```diff
@@ -83,6 +84,10 @@ func (s *bigtableObjectClient) PutChunks(ctx context.Context, chunks []chunk.Chu
 	return nil
 }
 
+func (s *bigtableObjectClient) PutWal(_ context.Context, _ *wal.SegmentWriter) error {
+	panic("not implemented")
```

Comment:
not very important as we won't use this code path, but we should probably just return an error rather than panicing (my bad!)